### ISSUE TITLE
Screen readers are not announcing property descriptions for BoolProperty CheckBoxes in project property pages when varying by configuration

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#8000]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 Preview
Version 17.2.0 Preview 1.0

## Summary
Screen readers are not announcing property descriptions for BoolProperty CheckBoxes in project property pages when varying by configuration.
- [Bug 1504564](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504564): A11y_Visual Studio Client_Code generation and run time section_LLVM_Info and relationships: Narrator/NVDA is not announcing the "Use the LLVM optimizing compiler" label name for the checkbox after selecting "Vary value by configuration" in the menu item

https://user-images.githubusercontent.com/59936622/159589789-0317bd98-e2c8-4556-bc14-65caac04bdb3.mp4

## Steps to Reproduce
1. Turn on Narrator/NVDA.
1. Create new iOS project from power shell.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "iOS Build" button and press enter.
1. Tab till "LLVM" section.
1. Tab till menu item and select "Vary value by configuration".
1. Check whether the screen reader is reading the label name for the checkbox or not.

## Expected Behavior
Narrator/NVDA should announce the "Use the LLVM optimizing compiler" label name for the checkbox after selecting "Vary value by configuration" in the menu item.

## Actual Behavior
Narrator/NVDA is not announcing the "Use the LLVM optimizing compiler" label name for the checkbox after selecting "Vary value by configuration" in the menu item.

## User Impact
Screen reader users may not be able to know the information about the checkbox field if screen reader does not announce the label names.

[MAS 1.3.1 Info and Relationships](https://liquid.microsoft.com/Web/Views/View/410630#Zrex-3A-2F-2Fms-2Eaccessibility-2FRequirements-2F01-2E03-2E01-2F)

**Comment by melytc** [comment]
@adamint this can be a great first a11y bug; the code where you can find the property descriptions is in CPS, happy to look at it together.

**Comment by adamint** [comment]
PR tracked in https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/395604
